app/graphicelement.h

```diff
@@ -10,9 +10,15 @@
 #include <QGraphicsPixmapItem>
 #include <QKeySequence>
 
+//enum class ElementType : uint_fast8_t {
+//  UNKNOWN, BUTTON, SWITCH, LED, NOT, AND, OR, NAND, NOR, CLOCK, XOR, XNOR, VCC, GND, DISPLAY,
+//  DLATCH, JKLATCH, DFLIPFLOP, JKFLIPFLOP, SRFLIPFLOP, TFLIPFLOP, TLATCH, BOX, NODE, MUX, DEMUX,
+//  BUZZER, DISPLAY14, LEDGRID
+//};
+
 enum class ElementType : uint_fast8_t {
   UNKNOWN, BUTTON, SWITCH, LED, NOT, AND, OR, NAND, NOR, CLOCK, XOR, XNOR, VCC, GND, DISPLAY,
-  DLATCH, JKLATCH, DFLIPFLOP, JKFLIPFLOP, SRFLIPFLOP, TFLIPFLOP, TLATCH, BOX, NODE, MUX, DEMUX,
```

Comment:
Caution! You should not change the sequence of the elements.
The enum value is used by the serialization process.
Older projects will break!

Please revert this change.

Reply:
One option to resolve this issue is to initialize each element with a number.
Another option is to keep the TLATCH enum value, and modify the factory method to return some sort of deprecation warning.